plugins/parquet/parquetembed.cpp

```diff
@@ -872,6 +872,21 @@ unsigned __int64 getUnsigned(std::shared_ptr<ParquetArrayVisitor> *array_visitor
     }
 }
 
+double getReal(std::shared_ptr<ParquetArrayVisitor> *array_visitor, int index)
```

Comment:
Minor: These functions could be static (to avoid the function names being exported globally).  Don't change for this PR though.